webpack/assets/javascripts/react_app/components/LoginPage/LoginPageHelpers.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { translate as __ } from '../../common/I18n';
+
+
+export const adjustProps = ({ token, version }) => ({
+  container: {
+    backgroundUrl: '/assets/login_background.jpg',
```

Comment:
i'm not sure hard coded url will work in production

Reply:
No, we have always had code to support remounting Foreman under a sub-uri. Not sure how well it still works, but I'd prefer to not break it. Assumptions around layout should be avoid IMHO.

Reply:
IIRC in production we also set expires headers and assume files do not change so that's another thing to consider here. Perhaps webpack already does some repacking and changes the URLs because AFAIK you can import files as well. Probably good to verify this and set up some guidelines for JS authors.